src/main/java/appeng/client/render/items/ToolBiometricCardRender.java

```diff
@@ -52,17 +52,18 @@ public boolean shouldUseRenderHelper( ItemRenderType type, ItemStack item, ItemR
 	@Override
 	public void renderItem( ItemRenderType type, ItemStack item, Object... data )
 	{
-		IIcon par2Icon = item.getIconIndex();
+		final IIcon par2Icon = item.getIconIndex();
 
-		float f4 = par2Icon.getMinU();
-		float f5 = par2Icon.getMaxU();
-		float f6 = par2Icon.getMinV();
-		float f7 = par2Icon.getMaxV();
-		float f12 = 0.0625F;
+		final float f4 = par2Icon.getMinU();
+		final float f5 = par2Icon.getMaxU();
+		final float f6 = par2Icon.getMinV();
+		final float f7 = par2Icon.getMaxV();
+		final float f12 = 0.0625F;
+
+		final Tessellator tessellator = Tessellator.instance;
 
-		Tessellator tessellator = Tessellator.instance;
 		GL11.glPushMatrix();
-		GL11.glPushAttrib( GL11.GL_ALL_ATTRIB_BITS );
+		// GL11.glPushAttrib( GL11.GL_ALL_ATTRIB_BITS );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)